src/Management/src/EndpointCore/AllActuatorsStartupFilter.cs

```diff
@@ -18,6 +20,21 @@ public AllActuatorsStartupFilter(Action<IEndpointConventionBuilder> configureCon
 
         private readonly Action<IEndpointConventionBuilder> _configureConventions;
 
+        public static void InitializeAvailability(IServiceProvider serviceProvider)
```

Comment:
Can this be internal? may be we shouldn't expose these at all  - if possible.

Reply:
This method should be available for anybody that doesn't want to use the Hostbuilder/StartupFilter approach, so it should be public... We could put it somewhere else if that would address your concern